tests/client/core/test_utils.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
Might be nice to disable this inside that block where we have a bunch of `of` loops then re-enable it, just so further rule violations don't crop up.